src/EnergyPlus/BoilerSteam.cc

```diff
@@ -519,6 +519,34 @@ namespace BoilerSteam {
+
+                        // Std 229 Boilers new report table
+                        OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchBoilerType, this->Name, "Boiler:Steam");
+                        OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchBoilerRefCap, this->Name, this->NomCap);
+                        OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchBoilerRefEff, this->Name, this->NomEffic);
+                        OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchBoilerRatedCap, this->Name, this->NomCap);
+                        OutputReportPredefined::PreDefTableEntry(state, state.dataOutRptPredefined->pdchBoilerRatedEff, this->Name, this->NomEffic);
+                        OutputReportPredefined::PreDefTableEntry(state,
+                                                                 state.dataOutRptPredefined->pdchBoilerPlantloopName,
+                                                                 this->Name,
+                                                                 this->plantLoc.loopNum > 0 ? state.dataPlnt->PlantLoop(this->plantLoc.loopNum).Name
+                                                                                            : "N/A");
+                        OutputReportPredefined::PreDefTableEntry(state,
+                                                                 state.dataOutRptPredefined->pdchBoilerPlantloopBranchName,
+                                                                 this->Name,
+                                                                 this->plantLoc.loopNum > 0 ? state.dataPlnt->PlantLoop(this->plantLoc.loopNum)
+                                                                                                  .LoopSide(this->plantLoc.loopSideNum)
+                                                                                                  .Branch(this->plantLoc.branchNum)
+                                                                                                  .Name
+                                                                                            : "N/A");
+                        OutputReportPredefined::PreDefTableEntry(
+                            state, state.dataOutRptPredefined->pdchBoilerMinPLR, this->Name, this->MinPartLoadRat);
+                        OutputReportPredefined::PreDefTableEntry(state,
+                                                                 state.dataOutRptPredefined->pdchBoilerFuelType,
+                                                                 this->Name,
+                                                                 Constant::eFuelNames[static_cast<int>(this->FuelType)]);
+                        OutputReportPredefined::PreDefTableEntry(
+                            state, state.dataOutRptPredefined->pdchBoilerParaElecLoad, this->Name, "Not Applicable");
```

Comment:
Wouldn't it be better to move these 2 big blocks of code, the one above and the one below, to below this big IF/ELSE block as 1 block of code protected with `if (state.dataPlnt->PlantFinalSizesOkayToReport) {`

Reply:
Thanks! That's seems to be a valid thought to reduce the code repetition. One little issue here is that the conditions for these two block do not seem to be perfectly aligned---the second one has an additional upper level condition to get into the blocks of `if (state.dataPlnt->PlantFinalSizesOkayToReport) {` as follows:
```
                    if (this->NomCap > 0.0 && tmpNomCap > 0.0) {
                        Real64 NomCapUser = this->NomCap;
                        if (state.dataPlnt->PlantFinalSizesOkayToReport) {
```

It is possible to combine these two added std 229 blocks into one, (literally) by adding a common condition to replicate the logic using `if(state.dataPlnt->PlantFinalSizesOkayToReport && (this->NomCapWasAutoSized || (!this->NomCapWasAutoSized && (this->NomCap > 0.0 && tmpNomCap > 0.0)))) {`. But it seems to somehow ruin the readability of the logic when doing this way. 

For this reason I would prefer the original version to keep the logic more readable (and also more closely attached to the original logic flow of doing the two `BaseSizer::reportSizerOutput()` branches.